less/window.less

```diff
@@ -6,9 +6,13 @@
 
 :root {
   --default-font-family: "Arial";
+  --zindex-frameWrapper: @zindexWindow;
+  --zindex-notActive: calc(@zindexWindow - 100);
+  --zindex-isPreview: calc(@zindexWindow + 100);
+  --zindex-fullScreen: @zindexFullScreen;
+  --zindex-fullScreenBanner: calc(@zindexFullScreen + 100);
```

Comment:
These aren't needed here. You only need this for things you want to access in JS as well.